app/models/ride/ride.ts

```diff
@@ -100,6 +100,9 @@ export const RideModel = types
         const supported = await iOSHelpers.canRunLiveActivities()
         this.setCanRunLiveActivities(supported)
         return supported
+      } else if (Platform.OS === "android") {
+        this.setCanRunLiveActivities(true)
+        return true
```

Comment:
girl how

Reply:
mm made it to resolve this ><
https://github.com/better-rail/app/blob/notifications-setup-screen/app/models/ride/ride.ts#L125

but yeah, that's incorrect

Reply:
just make sure it doesn't break anything. look where it's used (we don't want androids to get iOS alerts 😄)